### PR TITLE
refactor: add ref template user

### PR DESCRIPTION
**Pull request by John60676**
### 🤔 这个变动的性质是？/ What is the nature of this change?

- [ ] 新特性提交 / New feature
- [ ] bug 修复 / Fix bug
- [ ] 样式优化 / Style optimization
- [ ] 包体积优化 / Package size optimization
- [ ] 性能优化 / Performance optimization
- [ ] 代码风格优化 / Code style optimization
- [ ] 构建优化 / Build optimization
- [ ] 网站、文档、Demo 改进 / Website, documentation, demo improvements
- [ ] 测试相关 / Test related
- [x] 其他 / Other

### 📝 更新日志 / Changelog

| Language   | Changelog |
| ---------- | --------- |
| 🇺🇸 English |           |
| 🇨🇳 Chinese |           |

### ☑️ 请求合并前的自查清单 / Self Check before Merge

⚠️ 请自检并全部**勾选全部选项**。/ Please check all items below before review. ⚠️

- [x] 文档已补充或无须补充 / Doc is updated/provided or not needed
- [x] 代码演示已提供或无须提供 / Demo is updated/provided or not needed
- [x] TypeScript 定义已补充或无须补充 / TypeScript definition is updated/provided or not needed
- [x] Changelog 已提供或无须提供 / Changelog is provided or not needed

### CODE REVIEW

**Comment by xrkffgg** [comment]
rebase 下

**Comment by John60676** [comment]
done

File: README.md [hunk 1]
```diff
@@ -751,6 +751,13 @@ You can come to the following repositories for reference. Please leave a message
       <br/>
       <strong>ant-design-vue</strong>
     </td>
+    <td align="center">
+      <a href="https://github.com/AttoJS/vue-request">
+        <img src="https://raw.githubusercontent.com/AttoJS/art/master/vue-request-logo.png" width="46" />
```
Comment (xrkffgg) [inline README.md:756]: 你要用这个 logo 吗，我以为你会用 组织的图标呢

Comment (John60676) [reply README.md:756]: 哈哈哈，先用这个吧。logo 后面有空了还要重新设计一下

**Pull request merged**